tests/test_maker_utils.py

```diff
@@ -144,6 +144,9 @@ def test_override_data(node_class):
         The purpose here is just to inject unique non-default data into the maker utility to override
         its default values.
     """
+    # Skip the test if it is a SkycellsRef class
+    if node_class.__name__ == "SkycellsRef":
+        return
```

Comment:
```suggestion
```
We can test `SkycellsRef` here with a few other changes to the maker_utils functions added in this PR.

Reply:
Done.